### DISCUSSION TITLE
Electron, or CEF, or just customized Chromium.

### DISCUSSION THREAD

**Discussion by Alex313031** [radubirsan/FlashBrowser#13] [Q&A]
@radubirsan 
I know this uses npm to pack and run, so is it based on electron, CEF, or just chromium heavily modified?
Great work on the linux version. Using it happily.
Also, how come the 0.1 linux version has all the links go to a hentai porn site?

Also, I make my own fork of chromium for linux, win, and macos as well as builds of ChromiumOS for general PC's
https://github.com/Alex313031/Thorium/
https://github.com/Alex313031/Thorium-Win/
https://github.com/Alex313031/Thorium-Special/
https://github.com/Alex313031/ChromiumOS

Also, I extracted the .pngs from the .ico in the resources directory, and made a .desktop file for Flash Browser, I also appended --user-data-dir=$whereiputflashbrowser
Where does FlashBrowser normally store it's data?

You could unpack the icons as .pngs and modify this desktop file and include them in your repo for the linux builds
[FlashBrowser.desktop.zip](https://github.com/radubirsan/FlashBrowser/files/8146011/FlashBrowser.desktop.zip)

Lastly, can the libpepperflash.so be used on older chromium releases to enable flash?

**Comment by radubirsan** [comment]
@Alex313031  hi, thank you for your interest. the Linux build of FlashBrowser is out of date. 
I am looking someone to help us keeping it up to date. ( I am not sure how all the links go to hentai sites, it should point to flash.pm/games, we don't support porn )  
The project is using Electron with it's integrated Chromium engine. 
On Windows it stores it data C:\Users\{UserName}\AppData\Roaming\Flash Browser

The flash player used to be distributed for free but now I read some articles stating that you need a license.
I am still evaluating if there is still enough demand for a Flash dedicated browser to justify buying a license and launching a finished product.
